public/index.js

```diff
@@ -84,6 +84,14 @@ let errorMessage = `
 
 tryAgainBtn.style.display = "none";
 
+function createQuery(myObject) {
```

Comment:
This is your code from you latest un-push branch. It's just because I was updating my code on the latest branch and copied and pasted the code into a new branch. @josueJURE

Reply:
That might affect my number of contributions but no worries :-) lol